CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 3.5.0)
-set(CMAKE_CXX_FLAGS "-Wswitch -std=c++0x -pthread ${CMAKE_CXX_FLAGS} -Wno-undef")
+set(CMAKE_CXX_FLAGS "-Wswitch -std=c++0x -O2 -pthread ${CMAKE_CXX_FLAGS} -Wno-undef")
```

Comment:
I had a strange issue when adding this flag. Maybe it was caused by a bug that is already fixed, still better to check this carefully. I will need to do some local tests with the flag again

Reply:
okay, we could leave it out if it causes issues. 
i just thought it normally should be in..

Reply:
yes we can leave it out for this request. But I will try to make tests with the flag set still this week...